packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/index.js

```diff
@@ -55,6 +56,15 @@ const CellContent = ({ content, fieldSchema, metadatas, name, queryInfos, rowId
 
       return <SingleComponent value={content} metadatas={metadatas} />;
 
+    case 'string':
+      return (
+        <Tooltip description={<CellValue type={type} value={content} />}>
```

Comment:
This creates a prop-type error. Could you pass `content` to `description` instead?

```suggestion
        <Tooltip description={content}>
```